tsconfig.json

```diff
@@ -9,7 +9,8 @@
     "experimentalDecorators": true,
     "moduleResolution": "node",
     "importHelpers": true,
-    "target": "es2015",
+    "useDefineForClassFields": false,
+    "target": "es2022",
```

Comment:
I'm not sure we want to bump this to es2022 as that WILL cause potential issues for some projects using this extension.

Reply:
Those all updates are actually done by migrations and not manually.

Reply:
Yeh, that probably even more of a reason to delay this until we can create a specific version.